components/landing/LandingActions.tsx

```diff
@@ -16,6 +17,8 @@ const LandingActions: StylableFC = ({ style, className }) => {
 
   const plausible = usePlausible();
 
+  const [patchNotesOpen, setPatchNotesOpen] = useState(false);
```

Comment:
I see that you have implemented a useState to show/hide the patch notes. Very efficient